src/components/Select/Select.tsx

```diff
@@ -0,0 +1,315 @@
+import {
+  type HTMLAttributes,
+  type Ref,
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+} from "react";
+import { cva } from "../../../styled-system/css";
+import { Icon } from "../Icon/Icon";
+
+export interface SelectProps extends HTMLAttributes<HTMLSelectElement> {
```

Comment:
nitpick 1) import statement sorting 처럼, 우리가 직접 만든 prop과 기존 select element에서 가져온 prop의 순서를 구분하면 어떨까요? 그러면 헷갈리지 않고 좀 더 인터페이스를 빠르게 파악할 수 있을 거라 생각했어요.

```suggestion
export interface SelectProps extends HTMLAttributes<HTMLSelectElement> {
// custom props
// native props from HTMLSelectElement
```

Reply:
좋은 방법이네요 수정했습니다.